package.json

```diff
@@ -37,10 +37,10 @@
     "sharp": "0.33.2"
   },
   "devDependencies": {
-    "@captn/joy": "^0.20.0",
-    "@captn/react": "^0.20.0",
-    "@captn/theme": "^0.20.0",
-    "@captn/utils": "^0.20.0",
+    "@captn/joy": "^0.21.2",
+    "@captn/react": "^0.21.0",
+    "@captn/theme": "^0.21.0",
+    "@captn/utils": "^0.21.0",
```

Comment:
Is it correct, that only `@captn/joy` is `0.21.2` and all the others are still on `0.21.0`?

Reply:
Talked with @pixelass and this is fine.